tofu/tests/tests01_geom/test_01_GG.py

```diff
@@ -1797,16 +1797,16 @@ def test24_is_visible(debug=2):
             shot=0
         )
         s1 = tfg.PFC(Name="S1",
-                     Poly=[SP0x, SP0y],
+                     Poly=[SP0x[:-1], SP0y[:-1]],
```

Comment:
why these changes ? is it to leave the polygon open ?

Reply:
Yes, because the PFC class checks for doubles in the polygons.
If it provided as a closed polygon, it will work, but the class will throw a warning to the user just saying that it detected the double point.
It is a relatively normal behavior (double-checking inputs and keeping the user informed), but for unit tests we already know it and we just want to get rid of easy warnings so really problematic ones are more visible.